modules/document_repository/php/files.class.inc

```diff
@@ -381,7 +387,7 @@ class Files extends \NDB_Page
                         'version'       => $version,
                         'File_name'     => $fileName,
                         'File_size'     => $fileSize,
-                        'Data_dir'      => $fileName,
+                        'Data_dir'      => $uploadPath,
```

Comment:
I'm not sure that this will work with existing files in the document repository. You'll have entries like

`image.jpg` 
and 
`/data/loris/data/document_repository/admin/image.jpg`

in the same column

Reply:
fixed